cabal-install/src/Distribution/Client/Init/FileCreators.hs

```diff
@@ -67,12 +67,18 @@ writeProject (ProjectSettings opts pkgDesc libTarget exeTarget testTarget)
       exeStanza <- prepareExeTarget opts exeTarget
       testStanza <- prepareTestTarget opts testTarget
 
-      writeCabalFile opts $ pkgFields ++ [commonStanza, libStanza, exeStanza, testStanza]
+      (reusedCabal, cabalContents) <- writeCabalFile opts $
+        pkgFields ++ [commonStanza, libStanza, exeStanza, testStanza]
 
       when (null $ _pkgSynopsis pkgDesc) $
-        message opts T.Warning "no synopsis given. You should edit the .cabal file and add one."
+        message opts T.Warning "No synopsis given. You should edit the .cabal file and add one."
 
       message opts T.Info "You may want to edit the .cabal file and add a Description field."
+
+      when reusedCabal $ do
+        existingCabal <- readFile $ unPackageName (_optPkgName opts) ++ ".cabal"
+        when (existingCabal /= cabalContents) $
+          message opts T.Warning "The .cabal file was not updated, if that is desired please use the '--overwrite' option."
```

Comment:
It's not clear to me what is desired: updating or not updating. Perhaps "The .cabal file was found and was not updated. If updating is desired, ..."

Reply:
Done.

Reply:
I'd still be confused what is desired. :)

Reply:
"not updated, if that is desired" --- do you see?

Reply:
I see, now it's fixed.

Reply:
Perfect.